.github/workflows/github-actions-pr-checklist-check.yml

```diff
@@ -5,6 +5,7 @@ on:
 
 jobs:
   task-check:
+    permissions: write-all
```

Comment:
should it be here? :)

Reply:
this is a small fix for task-check for community :)

Reply:
moved to separate PR